plutus-tx/testlib/PlutusTx/Test/Golden.hs

```diff
@@ -169,3 +168,25 @@ goldenEvalCekLog
 goldenEvalCekLog name value =
   nestedGoldenVsDocM name ".eval" $
     prettyPlcClassicSimple . view _1 <$> rethrow (runPlcCekTrace value)
+
+prettyBudget :: PLC.ExBudget -> Size -> Doc ann
+prettyBudget (PLC.ExBudget (ExCPU cpu) (ExMemory mem)) (Size size) =
+  vsep
+    [ fill 8 "CPU:" <+> prettyIntRightAligned (fromSatInt @Int cpu)
+    , fill 8 "Memory:" <+> prettyIntRightAligned (fromSatInt @Int mem)
```

Comment:
What was the reason for the change to "CPU" and "Memory"? For consistency with `Pretty ExBudget` instance, I think we should stay with "cpu" and "mem".

https://github.com/IntersectMBO/plutus/blob/de1b3ef1a5677d8b636054cf4005a91dcc6cb733/plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudget.hs#L198-L202

Reply:
We should make the official names "CPU", "MEM" and "Size". The first two are metrics of script execution and they should look similar. The last one isn't and it's fine that it doesn't look alike.

Reply:
I won't block the PR over this, after all it's just aesthetic nitpicking.

This is my further reasoning for documentation purposes:
For me, CPU and Memory having different lengths made it feel more disconnected although they are both ExBudget and would always go together--this is unlike size which is outside of ExBudget.

Reply:
@SeungheonOh wanna shepherd that discussion? Move it to Slack, raise it during the technical meeting etc.